distributor-api-v1/changelog.md

```diff
@@ -1,5 +1,11 @@
 # Changelog
 
+## 26.02.2020
+
+* Added documentation for `hotels/getPaymentConfiguration`
+* Deleted old/legacy `payments/getPaymentGateway`
+* [PR](https://github.com/MewsSystems/gitbook-distributor-guide/pull/24)
```

Comment:
Not sure we should put this into public changelog. You can easily find this using blame.